keyboards/clawsome/gamebuddy/v1_M/config.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2021 AAClawson (AlisGraveNil)
```

Comment:
Folder name still needs to be lowercase.

Reply:
folder name adjusted from "v1_M" to "v1_m"